packages/aws-cdk-lib/core/test/stage.test.ts

```diff
@@ -252,36 +252,57 @@ describe('stage', () => {
     const rootAssembly = app.synth();
 
     // THEN
-    expect(rootAssembly.manifest.artifacts).toEqual({
+    expect(rootAssembly.manifest.artifacts).toMatchObject({
       'assembly-StageLevel1': {
         type: 'cdk:cloud-assembly',
         properties: {
           directoryName: 'assembly-StageLevel1',
           displayName: 'StageLevel1',
         },
       },
+      'aws-cdk-lib/feature-flag-report': {
```

Comment:
Don't need to test for this; `toMatchObject` will already allow additional artifacts to exist.